tests/offscreencanvas_tests.ts

```diff
@@ -29,9 +29,7 @@ function simpleTest(): void {
   // eslint-disable-next-line
   // @ts-ignore
   const offscreenCanvas = new OffscreenCanvas(550, 200);
-  // eslint-disable-next-line
-  // @ts-ignore
-  const offscreenCtx: OffscreenCanvasRenderingContext2D = offscreenCanvas.getContext('2d');
```

Comment:
Why were these removed? 

I think we had some build errors earlier. That's why I added a @ts-ignore here.

Removing the type name results in a build error in some instances.

Reply:
I do not remember how I got the complain. I have changed it now to cast on its usage to avoid any problem
```
const ctx = new CanvasContext(offscreenCtx as OffscreenCanvasRenderingContext2D);
```